Jenkinsfile

```diff
@@ -24,6 +24,7 @@ pipeline {
     PIPELINE_LOG_LEVEL = 'INFO'
     PYTEST_ADDOPTS = "${params.PYTEST_ADDOPTS}"
     RUNBLD_DISABLE_NOTIFICATIONS = 'true'
+    SLACK_CHANNEL = "#beats-ci-builds"
```

Comment:
Btw, we already have a `#beats-build` channel, could we use this one?

Reply:
Eventually, my idea was to play a bit with the notifications in a private channel, then once I'm happy with, I'll change it in another PR :)